src/assets/stylesheets/index.scss

```diff
@@ -1201,16 +1201,21 @@ section:focus >,
 /* ============ mobile view ============ */
 
 @media (max-width: 991px) {
-  .navbar-collapse {
+  #basic-navbar-nav.navbar-collapse {
     flex-flow: column-reverse;
   }
+
   .navbar-meta,
   .navbar-main {
     flex-flow: column;
 
     .navbar-nav {
       margin: 0 !important;
     }
+    &.navbar-nav {
+      flex-flow: column-reverse;
+    }
```

Comment:
Please try to position the search input at the very end of the list, or just above the "Current user" menu.

Reply:
Sorry, I didn't make it clear: The search input box should not be at the very end of _this_ list, but at the very end of navigation (main and meta combined). So either below "My Tynacoon" or (better) below "Current user info".

Reply:
Possible with markup change.

Reply:
Can you change the markup then?